java/src/org/openqa/selenium/safari/SafariDriverService.java

```diff
@@ -97,6 +103,10 @@ public int score(Capabilities capabilities) {
       return score;
     }
 
+    public Builder withOptions(SafariOptions options) {
```

Comment:
Just curious, I observed the options are passed but not used. Is that intentional?

Reply:
To make it consistent with the others. It's just to avoid telling everyone they need to use the `withOptions` method if they want to start a service and then say "except here". I'm fine to remove it if we want to go this route.